### DISCUSSION TITLE
RPI-RF-MOD: Externe Antenne benötigt? - Empfehlung?

### DISCUSSION THREAD

**Discussion by carusoe** [OpenCCU/OpenCCU#2061] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

ich habe mir meinen Raspi3 mit dem RPI-RF-MOD zusammengebaut und mittlerweile es auch geschafft RaspberryMatic als Plugin für den Home Assistant zum Laufen bekommen. Jetzt habe ich das Problem, dass mein HA/RaspberryMatic keine verlässliche Kommunikation zu den HM-Geräten (Heizungsthermostate & Türsensoren) hinbekommt. 
Jetzt meine Frage ist zweigeteilt:
1. Habt Ihr Tipps, wie ich die Kommunikation mit Boardmitteln verbessern kann?
2. Wenn eine externe Antenne notwendig nötig sein sollte, welche würdet Ihr empfehlen?

Viele Grüße
Carusoe

**Comment by jens-maus** [comment, marked as answer]
Auch wenn der Beitrag dazu schon etwas älter ist und sich dort auf das alte `HM-MOD-RPI-PCB` Funkmodul bezieht gilt es in gleichem Maße auch für das `RPI-RF-MOD`:

https://homematic-forum.de/forum/viewtopic.php?f=27&t=27287

D.h. eine Aurel Groundplane 868MHz Antenne direkt an das Funkmodul gelötet bzw. mit SMA-Adapter/Stecker steigert meiner Erfahrung nach die Sende-/Empfangsqualität des Funkmodules in starkem Maße.
